README.md

```diff
@@ -144,4 +146,34 @@ origin/aio/
 │   └── tutorial # チュートリアルのリソース
 ├── src # angular.ioアプリケーションのソースコード
 ...
-```
\ No newline at end of file
+```
+## Credits
+### Contributors
+
+This project exists thanks to all the people who contribute. 
+<a href="graphs/contributors"><img src="https://opencollective.com/angular-ja/contributors.svg?width=890&button=false" /></a>
```

Comment:
Where is the source of contributors list?
I visited OpenColllective page and these "contributors" section is empty.
is it calculated with GitHub repo contributors?